spec/features/two_factor_authentication/sign_in_spec.rb

```diff
@@ -267,27 +129,13 @@
     end
 
     scenario 'user attempts to circumnavigate OTP setup' do
-      second_factor = SecondFactor.find_by_name 'Email'
-      user = create(:user, :signed_up, second_factor_ids: second_factor.id)
+      user = create(:user, :signed_up)
```

Comment:
This test was not testing the right thing. Could we update it so that the user is not signed_up, and change the expectation to `expect(current_path).to eq users_otp_path`